libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/JSONUtils.java

```diff
@@ -25,6 +24,10 @@
      * traverse the object graph and pull out the desired property
      */
     public static <U> U queryJSON(JSONObject source, String query, U defaultObject) {
+        if (source == null) {
```

Comment:
> should we also check for a null query?

Yep, queries could come from an unitialized param added in 172378e

Unit Tests added in 63a24f8
 - can be run with: `./gradlew :libs:utils:WordPressUtils:connectedAndroidTest --offline`